### ISSUE TITLE
Error when running TransformFeedbackSmokeParticles

### ISSUE THREAD

**Issue by drewish** [cinder/Cinder#1008]
On OS X I'm getting this in the console:
```
|error  | cinder::gl::GlslProg::GlslProg(const cinder::gl::GlslProg::Format &)[602] Unknown attribute: "VertexVelocity"
|error  | cinder::gl::GlslProg::GlslProg(const cinder::gl::GlslProg::Format &)[602] Unknown attribute: "VertexInitialVelocity"
```

**Comment by ryanbartley** [comment]
hmm. It looks like the Velocity and InitialVelocity attributes were compiled out because of a change to the glsl or removed from the glsl, but they're still being defined in the GlslProg::Format. Has the glsl changed at all?

**Comment by andrewfb** [comment]
Should be fixed by ecf7f9e
